src/main/java/sirius/web/event/TemplateInvocationHandler.java

```diff
@@ -11,18 +11,18 @@
 import sirius.kernel.di.std.AutoRegister;
 
 /**
- * Represents a handler which handles a given URI.
+ * Represents a handler which handles template invocations.
  * <p>
  * This is used as a temporary solution in order to be able to log PageImpressionEvents for the HelpDispatcher.
```

Comment:
This comment is OX specific, I would drop it from here. If this is temporary and supposed to die, than it should be deprecated instead...